modules/node-management/pom.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It looks like it is not 'node-management' module, but just module with rest controller and some interfaces for it. Actual node management is doing under 'runner' module. 
If we create the separate top-level module for this kind of job - I fear we will be drowned in one-controller modules and become the REST factory instead of database :)

I agree, that maybe the runner is not the best place for it, but at the moment it is the real place where node lifecycle manages.

Maybe, if we have other controllers (which can't be placed in any existent modules), we'll should return to this topic and maybe create the separate place or package inside current 'rest*' modules. WDYT?

Reply:
The problem here is that `rest` module should know about `NodeManagementController` in order to generate the openapi specification at compile time. `NodeManagementController` depends on `LifecycleManager` from `runner` module.  And `runner` depends on `rest` module. Here is a cyclic dependency. So, I've added `node-management` in order to move all REST-related stuff out of `runner` module and solve the cyclic dependency problem. 

Your comment makes sense and I think there is a couple of ways we could go:
- choose a better name for the new module and/or move it under /rest/ path (make it a submodule of rest)
- move all stuff from `node-management` back to `runner` and try to solve the compile-time dependency between `runner` and `rest` in a different way. 

Any thoughts?